app/main.py

```diff
@@ -2,0 +1,27 @@
+import datetime
+from typing import List, Dict, Any
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: List[Dict[str, Any]], cafe: Cafe) -> str:
+    for friend in friends:
+        if "vaccine" not in friend:
+            return "All friends should be vaccinated"
+
+    not_vaccinated_count = sum(
+        1 for friend in friends
+        if friend["vaccine"]["expiration_date"] < datetime.date.today())
+    mask_needed_count = sum(
+        1 for friend in friends
+        if not friend.get("wearing_a_mask", True))
+
+    if not_vaccinated_count > 0:
+        return "All friends should be vaccinated"
+
+    if mask_needed_count > 0:
+        if mask_needed_count == 1:
+            return f"Friends should buy {mask_needed_count} mask"
+        else:
+            return f"Friends should buy {mask_needed_count} masks"
```

Comment:
Code Style issue: There is an unnecessary `else` statement on line 24. According to the checklist, we should avoid using `else` when it's not needed. In this case, you can simply return `Friends should buy {mask_needed_count} masks` after the `if` statement.